newsletter-bundle/src/Resources/contao/classes/Newsletter.php

```diff
@@ -339,6 +339,9 @@ protected function generateEmailObject(Database\Result $objNewsletter, $arrAttac
 			}
 		}
 
+		// Newsletters with an unsubscribe header are less likely to be blocked (see #2174)
+		$objEmail->addHeader('List-Unsubscribe', '<mailto:' . $objNewsletter->sender . '?subject=' . $GLOBALS['TL_LANG']['MSC']['unsubscribe'] . '>');
```

Comment:
```suggestion
		$objEmail->addHeader('List-Unsubscribe', '<mailto:' . $objNewsletter->sender . '?subject=' . rawurlencode($GLOBALS['TL_LANG']['MSC']['unsubscribe']) . '>');
```

We need to encode the subject I think because special characters in E-Mail headers might create problems (and the character `&` would end the subject).